jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -127,6 +128,20 @@ public PermissionConfiguration() {}
     }
   }
 
+  /** Configuration for {@code platform} parameter. */
+  public static class PlatformsConfiguration {
+    @Parameter private String os = "linux";
+    @Parameter private String architecture = "amd64";
+
+    public String getOs() {
```

Comment:
`getOs()` and `getArchitecture()` may not need to be `public`? I see some getters in other nested `-Configuration` classes don't have `public`. We can start with as much limited visibility as possible.

Reply:
We never use these methods locally so making them ` private ` throws ` unused  method warnings` .

Reply:
Java has so-called "package-private" in addition to `public`, `protected`, and `private`. Just remove the `public` keyword, and the method will also be accessible from any class in the same Java package.

Reply:
Nicee !! This is an interesting concept Java concept.
Done.